src/components/store/store-welcome/index.js

```diff
@@ -0,0 +1,57 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+
+import ROUTE from 'constants/route'
+import FooterButton from 'components/common/footer-button'
+import Icon from 'components/common/icon'
+
+import style from './index.style.js'
+
+export default class HomeScene extends React.Component {
+  render() {
+    return (
+      <div css={style.base}>
+        {this.renderTitle()}
+        <div css={style.composition}>
+          {this.renderBackground()}
+          {this.renderTag()}
+        </div>
+        {this.renderAddFirstItem()}
+      </div>
+    )
+  }
+
+  renderTitle() {
+    return (
+      <div css={style.title}>
+        <p css={style.title__text}>StoreName</p>
+        <p css={style.welcome__msg}>Welcome to your store!</p>
+      </div>
+    )
+  }
+
+  renderBackground() {
+    return (
+      <div css={style.background}>
+        <Icon.StoreArtBackground />
+      </div>
+    )
+  }
+
+  renderTag() {
+    return (
+      <div css={style.tag}>
+        <Icon.StoreArtTag />
+      </div>
+    )
+  }
+
+  renderAddFirstItem() {
+    return (
+      <div css={style.addFirstItem}>
```

Comment:
I made a `components/common/flex` component. I think we should try using it instead of `div`s in the future.

Reply:
done. works well.